src/routes/PlayRoute.js

```diff
@@ -7,16 +7,14 @@ import Header from "../components/modules/Header";
 import PageBody from "../components/common/PageBody";
 import Footer from "../components/modules/Footer";
 
-export default class PlayRoute extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      projects: [] // will be filled with entries in the form of:{name: <string>, description: <string>}
-    };
-    this.populateProjects();
-  }
+export default function PlayRoute() {
+  const [projects, setProjects] = React.useState([]);
+
+  React.useEffect(() => {
+    populateProjects();
+  }, []);
 
-  populateProjects() {
+  const populateProjects = () => {

```

Comment:
It looks like the body of this function could just be moved into the `useEffect` body?

Reply:
Moved function body to useEffect and replaced function name with descriptive comment. Commit 1d6c1b00b1bc65c27e636e2213d14634446bb669